src/backend/distributed/shared_library_init.c

```diff
@@ -916,6 +917,16 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_global_clock",
+		gettext_noop("Sets the cluster wide timestamp for transactions"),
```

Comment:
"Enables using get_cluster_clock() for the cluster-wide timestamp for transactions"